schemas/documents/shared/project.js

```diff
@@ -64,5 +65,10 @@ export default {
         list: ['hectares', 'acres'],
       },
     },
+    {
+      title: 'Compliance credits',
+      name: 'complianceCredits',
```

Comment:
seems like you forgot to add the new file for `complianceCredits`

Reply:
What do you mean by new file?
This is a new field in an existing file.

Reply:
oh sorry got confused between `name` and `type` fields, I thought you added a new `complianceCredits` type
because we need a new one, we need to store not only one number but 3 as you can see on https://www.figma.com/design/brkxGV5qNOkZUp0YQl1cxO/Terrasos-Phase-1?node-id=1117-70386&node-type=instance&m=dev: credits registered, available and retired

Reply:
Make sense, thanks for the explanation.

Reply:
I think it would be nice if their Sanity were a little cleaner.

Reply:
I have updated the structure to handle terrasos projects separately, so it will be cleaner for them. I'll update the code in the marketplace to handle that.